schemas/src/main/resources/profileschema/0.1/datahelix.schema.json

```diff
@@ -2,21 +2,26 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "http://scottlogic.com/datahelix.json",
   "definitions": {
+
     "schemaVersion": {
       "type": "string",
       "title": "The version of the DataHelix profile schema",
-      "pattern": "^[0-9]+.[0-9]+$"
+      "pattern": "^[0-9]+[.][0-9]+$"
```

Comment:
Probably just a prefence but it feels more standard to escape the any character symbol (`\\.`) rather than putting it in a character set.